.github/workflows/bot-executable.yml

```diff
@@ -0,0 +1,141 @@
+  build-bot:
+    needs: changes-detection
+    if: ${{ needs.changes-detection.outputs.bot == 'true' }}
+    runs-on: windows-latest
+
+    defaults:
+      run:
+        working-directory: ./runbotics/runbotics-desktop
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Clone submodules
+        run: git submodule update --init --recursive
+
+      - name: Install Node ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v2
+        with:
+          version: ${{ env.PNPM_VERSION }}
+
+      - name: Install dependencies
+        run:  ${{ env.RUSH }} install -t .
+
+      - name: Build
+        run: ${{ env.RUSH }} rebuild -t .
+
+      - name: Rush deploy
+        run: |
+          mkdir -p ../common/deploy/runbotics-desktop
+          ${{ env.RUSH }} deploy `
+            --scenario runbotics-desktop `
+            --target-folder ../common/deploy/runbotics-desktop `
+            --overwrite
+
+      - name: Assign package version
+        uses: nyaa8/package-version@v1
+        with:
+          path: './runbotics/runbotics-desktop/package.json'
+
+      - name: Install rb-sdk
+        run: |
+            pnpm install `
+              --ignore-scripts `
+              --dir `
+              ../common/deploy/runbotics-desktop/runbotics-sdk/
+
+      - name: Install rb-common
+        run: |
+          pnpm install `
+              --ignore-scripts `
+              --dir `
+              ../common/deploy/runbotics-desktop/runbotics-sdk/
+
+      - name: Copy .npmrc
+        run: |
+          cp `
+            ../common/deploy/runbotics-desktop/.npmrc `
+            ../common/deploy/runbotics-desktop/runbotics-desktop/
+
+      - name: Add rb-common and rb-sdk to pnpm
+        run: |
+          cd ../common/deploy/runbotics-desktop/runbotics-desktop/; `
+            pnpm add runbotics-common@file:../runbotics-common @runbotics/runbotics-sdk@file:../runbotics-sdk --ignore-scripts
+
+      - name: Run pnpm install
+        run: |
+          pnpm install `
+            --ignore-scripts `
+            --dir `
+            ../common/deploy/runbotics-desktop/runbotics-desktop/
+
+      - name: Set default sign to false
+        run: |
+          cd '../common/deploy/runbotics-desktop/runbotics-desktop/'; `
+          (Get-Content ./package.json -Raw | ConvertFrom-Json | ForEach-Object { if ($_.build -and $_.build.win) { $_.build.win | Add-Member -Name "verifyUpdateCodeSignature" -Value $false -MemberType NoteProperty } ; $_ } | ConvertTo-Json -Depth 20) | Set-Content ./package.json; `
+          (Get-Content ./package.json -Raw | ConvertFrom-Json | ForEach-Object { if ($_.build -and $_.build.win) { $_.build.win.PSObject.Properties.Remove('certificateFile'); $_.build.win.PSObject.Properties.Remove('certificatePassword') } ; $_ } | ConvertTo-Json -Depth 20) | Set-Content ./package.json
```

Comment:
what does it do?

Reply:
pnpm electron-builder does do the signing. It's not possible to skip this with any flag, so I have to change the package.json verifyUpdateCodeSignature to false and remove properties certificateFile and certificatePassword.

The action for signing does work only after running command pnpm electron-builder (aka after generating executable files)

Reply:
code signing does not work here? maybe we can skip if there is so much trouble with it